exonum/src/blockchain/schema.rs

```diff
@@ -48,27 +48,23 @@ define_names!(
     CONSENSUS_ROUND => "consensus_round";
 );
 
-encoding_struct! (
```

Comment:
this indentation looks weird

Reply:
maybe it wants {} braces?..

Reply:
I think we should submit a bug to `rustfmt`. Currently, I didn't find any issues related to this in their repository.

Reply:
> maybe it wants {} braces?..

I think this may be the case; see [the formatting RFCs](https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/statements.md#macros-in-statement-position):

> A macro use in statement position should use parentheses or square brackets as delimiters and should be terminated with a semi-colon.

So it's likely that the formatter thinks `encoding_struct!(...)` is a statement, not an item.